src/Compilers/Core/Portable/CodeAnalysisResources.resx

```diff
@@ -514,7 +514,7 @@
 {0}</value>
   </data>
   <data name="TupleNamesAllOrNone" xml:space="preserve">
```

Comment:
See https://github.com/dotnet/roslyn/pull/12595 for a nice way of naming these identifiers.  That way it is clear at the use site what the message will be.